main.tex

```diff
@@ -16,6 +16,9 @@
 \usepackage{verbatim}
 \usepackage{float}
 \usepackage{tikz}
+\usepackage{algpseudocode}
```

Comment:
oh, is this the substitute for:

`\usepackage[ruled, vlined]{algorithm2e}`

Reply:
? @walkeronwater  (let me tag you in case you didn't see it)

Reply:
oh ok
we are not writing the pseudocodes yet, so we can wait till we decide on that maybe?

Reply:
Yeah definitely fine. I'll do the change if needed

Reply:
:) thanks !

Reply:
so, can we remove this line for now? 
Line 19 --> to make sure that we don't face any package conflicts or stuff like that.

sorry for the back-and-forth communication

Reply:
removed the packages (sorry for the delay)